run-docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+./x11docker/x11docker --gpu --runtime=nvidia --network=none --verbose -- \
+-v ${HOME}/webots_video_out/:/out \
+-v $(pwd)/teams/${TEAM_Y}/bot1/bot1.py:/app/controllers/rcj_soccer_player_y1/rcj_soccer_player_y1.py:ro \
```

Comment:
@dbscoach I'd very much prefer us map the folders rather the specific Python scripts, just for the simplicity of it :)

Reply:
We had already named the test files bot1/bot1.py etc. I'm not married to the concept, but this way I can already run two halves correctly. I'll adapt to whatever you specify for submission. I changed the name of the PR from "Implementation" to "Proof of Concept", none of this needs to stay.

Reply:
Thanks @dbscoach, it's all good, please do not read this in any other way than "I am extremely happy this even works and would want to thinker more with it :slightly_frowning_face: